lib/spack/spack/schema/cdash.py

```diff
@@ -15,7 +15,8 @@
     "cdash": {
         "type": "object",
         "additionalProperties": False,
-        "required": ["build-group", "url", "project", "site"],
+        # "required": ["build-group", "url", "project", "site"],
```

Comment:
We probably don't want this comment in here anymore.

Reply:
This is an issue that arose because part of cdash is specified in one config scope, and the rest of the config is in the stack `spack.yaml` They are all ultimately required.